src/main/app.ts

```diff
@@ -32,6 +33,9 @@ new Nunjucks(developmentMode).enableFor(app);
 // secure the application by adding various HTTP headers to its responses
 new Helmet(config.get('security')).enableFor(app);
 
+axios.defaults.baseURL = config.get('ardoq.apiUrl');
```

Comment:
sorry to nit pick but rather than setting defaults on the global axios you might want to create an instance with the defaults